src/Tizen.Wearable.CircularUI.Forms/CircularShell.cs

```diff
@@ -2,43 +2,83 @@
+
+        /// <summary>
+        /// BindableProperty. Identifies the FlyoutForegroundColor bindable property.
+        /// </summary>
         public static readonly BindableProperty FlyoutForegroundColorProperty = BindableProperty.CreateAttached("FlyoutForegroundColor", typeof(Color), typeof(Shell), Color.Default);
 
+        /// <summary>
+        /// Get a color of FlyoutIconBackgroundColor
+        /// </summary>
+        /// <param name="element">Xamarin.Forms.Shell</param>
+        /// <returns>Returns a Color of FlyoutIconBackgroundColor</returns>
         public static Color GetFlyoutIconBackgroundColor(BindableObject element)
         {
             return (Color)element.GetValue(FlyoutIconBackgroundColorProperty);
         }
 
+        /// <summary>
+        /// Set a color of FlyoutIconBackgroundColor
+        /// </summary>
+        /// <param name="element">Xamarin.Forms.Shell</param>
+        /// <param name="color">Xamarin.Forms.Color</param>
         public static void SetFlyoutIconBackgroundColor(BindableObject element, Color color)
         {
             element.SetValue(FlyoutIconBackgroundColorProperty, color);
         }
 
+        /// <summary>
+        /// Get a color of FlyoutForegroundColor
+        /// </summary>
+        /// <param name="element">Xamarin.Forms.Shell</param>
+        /// <returns>Returns a Color of FlyoutForegroundColor</returns>
         public static Color GetFlyoutForegroundColor(BindableObject element)
         {
             return (Color)element.GetValue(FlyoutForegroundColorProperty);
         }
 
+        /// <summary>
+        /// Set a color of FlyoutForegroundColor
+        /// </summary>
+        /// <param name="element">Xamarin.Forms.Shell</param>
+        /// <param name="color">Xamarin.Forms.Color</param>
         public static void SetFlyoutForegroundColor(BindableObject element, Color color)
         {
             element.SetValue(FlyoutForegroundColorProperty, color);
         }
 
+        /// <summary>
+        /// Gets or sets FlyoutIconBackgroundColor
+        /// </summary>
         public Color FlyoutIconBackgroundColor
         {
             get => (Color)GetValue(FlyoutIconBackgroundColorProperty);
             set => SetValue(FlyoutIconBackgroundColorProperty, value);
         }
 
+        /// <summary>
+        /// Gets or sets FlyoutForegroundColor
+        /// </summary>
         public Color FlyoutForegroundColor
         {
             get => (Color)GetValue(FlyoutForegroundColorProperty);
             set => SetValue(FlyoutForegroundColorProperty, value);
         }
 
+        /// <summary>
+        /// Callback that redefines the action when the device's back button is pressed.
+        /// </summary>
+        /// <returns>Basically application is closed and return false, but if the flyout is presented, close the flyout and return true.</returns>
```

Comment:
<returns>When this method return true, application will not be terminated, otherwise it will be terminated.</returns>
<remarks> The default behavior of CircularShell when Flyout is opened is to close the Flyout without terminating an application.
 </remarks>

Reply:
ok